BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1842,6 +1878,9 @@ void EndBoost()
 
         IEnumerator CruiseRoutine()
         {
+            burnRate = (cruiseTime == 0 || sustainerMass < 0) ? 0 : (sustainerMass / cruiseTime) * Time.fixedDeltaTime; //Same as in booster
```

Comment:
If you set `boosterMass` and `sustainerMass` to 0 in `OnStart` if they're negative or they're non-zero while `boostTime` and `cruiseTime` are 0 respectively (in which case, log a warning about badly configured missiles), then these statements simplify to
```csharp
burnRate = sustainerMass > 0 ? sustainerMass / cruiseTime * Time.fixedDeltaTime : 0; // Same as in booster
burnedMass = boosterMass; // This shouldn't be set if not all the booster mass is used.
float massToBurn = burnedMass + sustainerMass;
```